metric/bigtable/src/main/java/com/spotify/heroic/metric/bigtable/BigtableBackend.java

```diff
@@ -472,8 +472,12 @@ public Statistics getStatistics() {
             fetches.add(readRows.directTransform(result -> {
                 for (final FlatRow row : result) {
                     watcher.readData(row.getCells().size());
-                    List<Metric> metrics = Lists.transform(row.getCells(), transform);
+                    final List<Metric> metrics = Lists.transform(row.getCells(), transform);
                     metricsConsumer.accept(MetricCollection.build(type, metrics));
+
+                    if (metrics.size() > 1) {
+                        watcher.reportRowDensity(getDensity(metrics));
```

Comment:
1. Good point, I believe the code is there because of earlier reasons while I was experimenting - reasons that aren't valid any more. I'll move it to ResultCollector in LocalMetricManager.
1. metrics.size() == 0: So, throw an error and fail the whole read operation or add an entry to requestErrors?
1. metrics.size() == 1: How about this case? Can't calculate the density (distance between metrics) when there's only one metric. Is this a problem?